lib/honeybee/model_object.rb

```diff
@@ -64,7 +65,8 @@ def method_missing(sym, *args)
     def self.read_from_disk(file)
       hash = nil
       File.open(File.join(file), 'r') do |f|
-        hash = JSON.parse(f.read, symbolize_names: true)
+        hash = JSON.parse(File.read(f, :external_encoding => 'iso-8859-1',
+          :internal_encoding => 'utf-8'), symbolize_names: true, encoding: 'UTF-8')
```

Comment:
This helped - when I print this hash I see Unicode characters. Somehow this is not translating to the object classes hash (https://github.com/ladybug-tools/honeybee-openstudio-gem/pull/245/files#diff-2296abac85df1839aba12ad4a880f93c287d91392eacc4c21cbfea9cfe9c5949R53)

Reply:
This looks like you are on the right track.  Is this reproducible on a local WIndows machine or does it only fail on CI?

Reply:
@macumber you can reproduce it by uncommenting this test: https://github.com/ladybug-tools/honeybee-openstudio-gem/pull/245/files#diff-2296abac85df1839aba12ad4a880f93c287d91392eacc4c21cbfea9cfe9c5949R53

Reply:
Try this out: https://github.com/tanushree04/honeybee-openstudio-gem/pull/1

Reply:
@macumber this worked for me. Thanks for all your help!

Reply:
I merged your PR to this.